cmd/service/init.go

```diff
@@ -140,12 +141,19 @@ func getTemplateResult(result string, templates []*templateStruct) (tmpl *templa
 }
 
 func downloadTemplate(tmpl *templateStruct) (path string, err error) {
-	path, err = createTempFolder()
+	path, err = ioutil.TempDir("", utils.TempDirPrefix)
 	if err != nil {
 		return "", err
 	}
 
-	return path, gitClone(tmpl.URL, path, "Downloading template "+tmpl.Name+"...")
+	message := fmt.Sprintf("Downloading template %s ...\n", tmpl.Name)
```

Comment:
This is not valid, the `\n` create some problems with the spinner. We should never use a `\n` when it's a message for the spinner
```
⣾ Downloading template Javascript ...
⣽ Downloading template Javascript ...
⣻ Downloading template Javascript ...
⢿ Downloading template Javascript ...
⡿ Downloading template Javascript ...
```

Reply:
Done